pkgs/development/python-modules/huepy/default.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchPypi,
+  buildPythonPackage,
+  pytest,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "huepy";
+  version = "1.2.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "huepy";
+    inherit version;
+    hash = "sha256-Wym+73lzEvt2BhiLxc2Y94q49+AVdkJ6kxLxybILdZ0=";
+  };
+
+  nativeBuildInputs = [ setuptools];
```

I believe this is the recommended attribute for python build time dependencies.
See https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

```suggestion
  build-system = [ setuptools ];
```